src/locales/fi.json

```diff
@@ -607,12 +603,17 @@
       },
       "title": "Kiitos!"
     },
+    "payment_failed": {
+      "title": "Virhetilanne",
+      "paragraph1": "Pahoittelemme, mutta maksun käsittelyssä tapahtui virhe.",
+      "paragraph2": "Voit yrittää maksua myöhemmin uudelleen, tarvittaessa ota yhteyttä asiakaspalveluun."
+    },
     "payment": {
       "title": "Laskun maksaminen",
-      "terms_info": "Hyvä asiakkaamme, olemme päivittäneet venepaikan sopimusehtoja. Tutustu sopimusehtojen sisältöön alla. Voit jatkaa maksutapahtumaan sen jälkeen. Venepaikan saaminen edellyttää sopimusehtojen hyväksymistä.",
+      "terms_info": "Hyvä asiakkaamme, olemme päivittäneet talvisäilytyspaikan sopimusehtoja. Tutustu sopimusehtojen sisältöön alla. Voit jatkaa maksutapahtumaan sen jälkeen. Talvisäilytyspaikan saaminen edellyttää sopimusehtojen hyväksymistä.",
       "questions": "Kysymykset ja lisätiedot: ",
-      "terms_pdf": "Venepaikan sopimusehdot (PDF)",
-      "accept_terms": "Olen lukenut venepaikan sopimusehdot ja hyväksyn ne",
+      "terms_pdf": "Talvisäilytyspaikan sopimusehdot (PDF)",
+      "accept_terms": "Olen lukenut talvisäilytyspaikan sopimusehdot ja hyväksyn ne",
       "pay": "Jatka maksamaan"
     },
```

Comment:
Is this because the first payments will be winter storage related, and the berth payments will be developed later (with changes based on how the winter storage season goes)?

Reply:
Yes